CODE_OF_CONDUCT.md

```diff
@@ -10,30 +8,43 @@ We pledge to act and interact in ways that contribute to an open, welcoming, div
-* Private harassment is also unacceptable. No matter who you are, if you feel you have been or are being harassed or made uncomfortable by a community member, please contact the maintainers immediately. Whether you’re a regular contributor or a newcomer, we care about making this community a safe place for you and we’ve got your back.
-* Do not make casual mention of slavery or indentured servitude and/or false comparisons of one's occupation or situation to slavery. Please consider using or asking about alternate terminology when referring to such metaphors in technology.
-* Likewise any spamming, trolling, flaming, baiting or other attention-stealing behavior is not welcome.
+Respect that people have differences of opinion and that every design or implementation choice carries a trade-off and numerous costs. There is seldom a right answer.
+Bevy welcomes experimentation and iteration: if you have an idea for how something could be done better, you want to experiment with, make a PR or write an RFC to show us how it compares in practice!
 
 ## Moderation
 
 These are the policies for upholding [our community’s standards of conduct](#our-standards). If you feel that a thread needs moderation, please contact the maintainers.
-
-1. Remarks that violate the community standards of conduct, including hateful, hurtful, oppressive, or exclusionary remarks, are not allowed. (Cursing is allowed, but never targeting another user, and never in a hateful manner).
-2. Remarks that maintainers find inappropriate, whether listed in the code of conduct or not, are also not allowed.
-3. Maintainers will first respond to such remarks with a warning.
-4. If the warning is unheeded, the user will be “kicked,” i.e., kicked out of the communication channel to cool off.
-5. If the user comes back and continues to make trouble, they will be banned, i.e., indefinitely excluded.
-6. Maintainers may choose at their discretion to un-ban the user if it was a first offense and they offer the offended party a genuine apology.
-7. If a maintainer bans someone and you think it was unjustified, please take it up with that maintainer, or with a different maintainer, in private. Complaints about bans in-channel are not allowed.
-8. Maintainers are held to a higher standard than other community members. If a maintainer creates an inappropriate situation, they should expect less leeway than others.
-
 The enforcement policies in the code of conduct apply to all official venues, including Discord channels, GitHub repositories, the Twitter community and all other forums.
+
+1. Hateful, hurtful, oppressive, or exclusionary remarks are not allowed.
+2. Personal attacks are out-of-line.
+   1. Disagree with specific actions and decisions in a constructive way.
```

Comment:
Having moderated a few communities myself, I've long preferred short-form rule wording so that it's easier to get the full breadth. Could this sub bullet points be shortened and merged with the top level ones? or omitted entirely? A lot of these are "common sense" justifications or alternatives.